src/providers/JsonRpcProvider.js

```diff
@@ -5,12 +5,15 @@ import { get, has } from 'lodash'
 import Provider from '../Provider'
 import RpcError from './RpcError'
 
+import networks from '../networks'
+
 const { parse } = JSONBigInt({ storeAsString: true, strict: true })
 
 export default class JsonRpcProvider extends Provider {
-  constructor (uri, username, password) {
+  constructor (uri, username, password, chain = { network: networks.bitcoin }) {
```

Comment:
This is an abstract base class. is it possible to move this to BitcoinRPCProvider?